addon/services/fb.js

```diff
@@ -49,27 +49,9 @@ export default Ember.Service.extend({
     this.setAccessToken(token);
   },
 
-  api(path) {
-    var method = 'GET';
-    var parameters = {};
-    var arg;
-
+  api(path, method = 'GET', parameters = {}) {
```

Comment:
It's not 100% correct. For example, if I call `api('/me', {'foo': 'bar'})` it would issue a call to the Facebook API with `GET` method and `parameters` `{'foo': 'bar'}`